src/components/Modal/DiscountModal.style.ts

```diff
@@ -0,0 +1,54 @@
+import styled from 'styled-components'
+
+export const ModalBackground = styled.div`
+  position: absolute;
+  top: 0;
+  left: 0;
+  width: 100%;
+  height: 100%;
+  background: rgba(0, 0, 0, 0.5);
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  align-items: center;
+  z-index: 1000;
+`
+
+export const ModalTextWrapper = styled.div`
+  font-size: 1rem;
+  color: #333;
+  margin: 0 0 10px;
+  text-align: center;
+  font-weight: bold;
+  line-height: 0.5;
+
+  &:last-child {
+    margin-bottom: 0;
+  }
+`
+
+export const ModalContainer = styled.div`
+  display: flex;
+  flex-direction: column;
+  justify-content: space-evenly;
+  align-items: center;
+  width: 252px;
+  height: 300px;
+  background: white;
+  border-radius: 20px;
```

Comment:
얘는 border-radius 15px입니당! border-radius 집착녀 같네요.. ㅋㅋㅋ
```suggestion
  border-radius: 15px;
```

Reply:
radius 오오 디테일한거 봐주셔서 감사하죠 ㅎㅎ 고쳐놓고 다시 푸쉬할게요!